app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -69,10 +69,10 @@
       <span id="pageviews-<%= article.id %>" class="cta pill dashboard-pageviews-indicator" data-analytics-pageviews data-article-id="<%= article.id %>">
         <span class="page-views-count">
           <span class="value">
-            <% if article.page_views_count > 100 %>
```

Comment:
Thought: let's move this to `SiteConfig` so we don't need to make another PR if this ever changes? Magic numbers in the code are not great IMHO.

```ruby
<% if article.page_views_count > SiteConfig.min_page_view_counts %>
# and
under <%= SiteConfig.min_page_view_counts %>
```

If you need some help with `SiteConfig` feel free to ping me.

Reply:
I'm not sure it's worth changing right now, as we might want to revisit and regroup some configuration options in a larger schema of generalization. I'm a bit wary of the fact that we might end up adding values to `SiteConfig` without a master plan :D

Reply:
Works for me. Maybe keep track of it somewhere though, so we revisit it in the future.